### ISSUE TITLE
register in multiple gitlab-ci hosts

### ISSUE THREAD

**Issue by fernandoneto** [sameersbn/docker-gitlab-ci-multi-runner#9]
@sameersbn do you know if it is possible to register 1 multi-runner in multiple gitlab-ci containers?
in this moment i have 2 gitlab containers sharing the `/home/git/data` folder, but when i check the gitlab registration token they are different in the 2 container's.

**Comment by sameersbn** [comment]
I don't think it is possible. But i am not sure. You will need to check with the upstream project

**Comment by fernandoneto** [comment]
just to keep the tracking if someone else have the same problem

https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/issues/250
